access/pagerduty/pagerduty_test.go

```diff
@@ -624,7 +624,7 @@ func (s *PagerdutySuite) assertReviewSubmitted() {
 	require.NoError(t, err)
 	defer watcher.Close()
 
-	_ = s.assertNewEvent(watcher, types.OpInit, "", "")
+	_ = s.assertNewEvent(watcher, types.OpInit, types.KindWatchStatus, "watch-status")
```

Comment:
`OpInit` now returns a resource: `WatchStatus`
This was changed in RFD114 https://github.com/gravitational/teleport/blob/master/rfd/0114-partial-cache-healthiness.md